cypress/integration/creatorOnboardingFlows/creatorSignup.spec.js

```diff
@@ -71,7 +71,7 @@ describe('Creator Signup Page', () => {
     cy.get('.js-eye').should('be.visible');
     cy.get('.js-eye-off').should('not.be.visible');
```

Comment:
Once the `aria-pressed` attribute has been added to the toggle button, it would be cool if we could also test 

`cy.findByRole('button', { name: 'Show password' }).should('have.attr', 'aria-pressed', 'true')` 😍